samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/cs/source2.cs

```diff
@@ -80,13 +80,13 @@ class SameVehicleComparer : EqualityComparer<string>
 {
     public override bool Equals(string s1, string s2)
     {
-        return s1.Equals(s2, StringComparison.CurrentCultureIgnoreCase);
+        return string.Equals(s1, s2, StringComparison.OrdinalIgnoreCase);
```

Comment:
Wouldn't `StringComparer.OrdinalIgnoreCase.Equals(s1, s2)` be simpler and more logical here?  This class basically duplicates StringComparer.OrdinalIgnoreCase.

Reply:
That's an excellent point; I'm not sure if the documentation is trying to show *how* to write an equality comparer, or simply how to use a custom one. I think showing how to use `StringComparer.OrdinalIgnoreCase` is the better approach; will push an update.